openid-4-verifiable-presentations-1_0.md

```diff
@@ -828,40 +828,17 @@ The following is a non-normative example of a set of static configuration values
 }
 ```
 
-## Support for Federations/Trust Schemes
+## Support for Federations and Trust Schemes
```

Comment:
I'd change Trust Schemes to Trust Models if you agree

I heard some comments about the concrete meaning of the term trust scheme

Reply:
why not trust framework? what is the distinction/definition of trust scheme vs trust model vs trust framework?

Reply:
I'm in favor of Trust Framework, I'm doing this change

Reply:
Done c72b02f0791d0de05d5428a39c3672b1e4f58636